docs/relational-databases/security/authentication-access/server-level-roles.md

```diff
@@ -91,7 +91,7 @@ The following table shows the permissions assigned to the server-level roles. It
 | --- | --- | --- |
 | `##MS_DatabaseConnector##` | - `CONNECT ANY DATABASE` | - `CONNECT` |
 | `##MS_LoginManager##` | - `CREATE LOGIN`<br />- `ALTER ANY LOGIN` | N/A |
-| `##MS_DatabaseManager##` | - `CREATE ANY DATABASE`<br />- `ALTER ANY DATABASE` | - `ALTER` |
+| `##MS_DatabaseManager##` | - `CREATE ANY DATABASE`<br />- `ALTER ANY DATABASE` |  `DROP ANY DATABASE` - `ALTER` |
```

Comment:
```suggestion
| `##MS_DatabaseManager##` | - `CREATE ANY DATABASE`<br />- `ALTER ANY DATABASE`<br />- `DROP ANY DATABASE` | - `ALTER` |
```

`ALTER ANY DATABASE` already implies that the role has `DROP ANY DATABASE` permissions. I don't think this needs to be added, but I did fix the formatting. https://learn.microsoft.com/en-us/sql/t-sql/statements/drop-database-transact-sql?view=sql-server-ver16#permissions

@rwestMSFT - Can I please get your opinion on this? Thanks!

Reply:
Agreed. It's implied from context.